v3/rfcs/dx/20240620-simplify-subgraph-getting-started-and-federation.md

```diff
@@ -0,0 +1,47 @@
+# DX: simplify subgraphs for getting started and federation workflows
+
+Status: Review required
+
+## subgraph init command
+What this is: as a part of making it easier for people to work through the getting started guide, or to scaffold project files, allow the user to go from scratch to an API with only CLI commands.
+
+init-ing a supergraph or connector will create files for both local and cloud environments. This is about what happens for the subgraphs.
+
+```bash
+# creates a subgraph folder and subgraph.yaml file
+ddn subgraph init app
```

Comment:
Is this for a federation use case, where we create a standalone subgraph? If it is not, the intention of this command is to not add the `subgraph.yaml` to add `supergraph.yaml`. But if the user has set `supergraph` in context, we will have to use it, which might not be the case here. To avoid this confusion, can we change the flag to `--add-to-supergraph`?